src/common/components/Button/Button.theme.ts

```diff
@@ -127,6 +127,18 @@ export const buttonTheme = tv(
           "[&>*:not(.loading)]:invisible",
         ],
       },
+      rounded: {
+        true: [
+          "inline-flex",
+          "min-w-14",
+          "py-[0.125rem]",
+          "px-3",
+          "gap-2",
+          "rounded-[6.1875rem]",
+          "border-solid",
+          "border-border-default",
```

Comment:
Somehow the hover bg doesn't look as rounded as the default state 

https://github.com/AfterClass-io/afterclass.io-v2/assets/6003064/7b861157-4be0-48b9-987a-f1e7ef291891

Reply:
great catch! fixed at f4f6aec